reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/SwitchIfEmpty.kt

```diff
@@ -0,0 +1,52 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.base.ErrorCallback
+import com.badoo.reaktive.disposable.Disposable
+import com.badoo.reaktive.disposable.DisposableWrapper
+import com.badoo.reaktive.utils.atomicreference.AtomicReference
+
+fun <T> Observable<T>.switchIfEmpty(otherObservable: Observable<T>): Observable<T> =
+    switchIfEmpty { otherObservable }
+
+fun <T> Observable<T>.switchIfEmpty(otherObservable: () -> Observable<T>): Observable<T> =
+    observableUnsafe { observer ->
```

Comment:
In this case it's better to use `observable { emitter ->` as if `otherObservable` will throw an exception and upstream will emit an item, then this item will be emitted to downstream (emission after error). Emitter is safe and will protect from such cases.